src/Telemetry/AzureDevOpsClient/AzureDevOpsClientFactory.cs

```diff
@@ -10,9 +11,20 @@ public sealed class AzureDevOpsClientFactory : IAzureDevOpsClientFactory
     {
         public AzureDevOpsClientFactory() { }
 
-        public IAzureDevOpsClient CreateAzureDevOpsClient(string baseUrl, string organization, int maxParallelRequests, string accessToken)
+        public IAzureDevOpsClient CreateAzureDevOpsClient(
+            string baseUrl,
+            string organization,
+            int maxParallelRequests,
+            string accessToken,
+            IHttpClientFactory clientFactory)
```

Comment:
Why can't this class take this in the construtor rather than adding another parameter? This type is already set up for DI (no one is constructing one directly). And this is the only method in it, so it's not like someone will try to use it without HTTP.

Reply:
Same thing for the options, honestly, but I wouldn't mind leaving those. (Someone might want to ask for something in a different organization, like devdiv, for example, especially since we are looking at rearranging our project structure)

Reply:
I injected the HttpClientFactory